packages/client/src/offline/reducer.ts

```diff
@@ -105,11 +105,7 @@ export interface IOfflineData {
     fonts?: CertificateConfiguration['fonts']
     // Certificates might not be defined in the case of
     // a field agent using the app.
-    certificates?: {
-      birth: ISVGTemplate
-      death: ISVGTemplate
-      marriage: ISVGTemplate
-    }
+    certificates?: ICertificateConfigData[]
```

Comment:
Is the above comment still true? Should all template configs be available regardless to all users. That would help you further on in the code as we wouldn't have to keep on checking if they are available or not. 
If the answer is, no, we can't show field agents the config then let's at least default this to an empty list so you can trust the list always exists

Reply:
The field agents don't have the access to certificates.